pkg/controller/kogitoinfra/infinispan/infinispan_properties.go

```diff
@@ -41,6 +43,8 @@ const (
 	envVarInfinispanPassword
 	infinispanEnvKeyCredSecret = "INFINISPAN_CREDENTIAL_SECRET"
 	enablePersistenceEnvKey    = "ENABLE_PERSISTENCE"
+	// SASLPlain is the PLAIN type.
+	SASLPlain string = "PLAIN"
```

Comment:
```suggestion
	saslPlain string = "PLAIN"
```

We don't need to export it, do we?

Reply:
done